lib/util/content.py

```diff
@@ -12,54 +12,62 @@
     user_content = Path(user_content)
 
 
-def find_datastream_in(root):
-    base_dir = root / Path('usr/share/xml/scap/ssg/content')
+def _find_datastreams(root='/'):
+    if user_content:
+        build_content(user_content)
+        datastreams_dir = user_content / 'build'
+    else:
+        datastreams_dir = root / Path('usr/share/xml/scap/ssg/content')
+    return datastreams_dir
+
+
+def get_datastream(root='/'):
     if rhel.is_true_rhel():
-        return base_dir / f'ssg-rhel{rhel.major}-ds.xml'
+        name = f'ssg-rhel{rhel.major}-ds.xml'
     elif rhel.is_centos():
         if rhel <= 8:
-            return base_dir / f'ssg-centos{rhel.major}-ds.xml'
+            name = f'ssg-centos{rhel.major}-ds.xml'
         else:
-            return base_dir / f'ssg-cs{rhel.major}-ds.xml'
-
-
-def get_datastream():
-    if user_content:
-        build_content(user_content)
-        datastream = user_content / 'build' / f'ssg-rhel{rhel.major}-ds.xml'
-    else:
-        datastream = find_datastream_in('/')
+            name = f'ssg-cs{rhel.major}-ds.xml'
+    datastream = _find_datastreams() / name
```

Comment:
This should probably call `_find_datastreams(root)`.

Thinking about it, `_find_datastreams()` should probably have that argument mandatory (without a default), as it's not meant to be used externally. Any default will be sorted out by `get_datastream()` already.

Same for `_find_playbooks()`.